.eslintrc.cjs

```diff
@@ -13,7 +13,7 @@ module.exports = {
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-var-requires": "off",
     "@typescript-eslint/no-this-alias": "off",
-    "no-console": "warn",
+    "no-console": ["error", { allow: ["warn", "error"] }],
```

Comment:
I changed this because using `console.error` or `warn` isn't necessarily invalid in the context of this project. Right now both usages of `console.log` could be removed so this shouldn't be a problem